### ISSUE TITLE
jsx-a11y/control-has-associated-label issuing warnings for hidden fields

### ISSUE THREAD

**Issue by hades200082** [jsx-eslint/eslint-plugin-jsx-a11y#691]
```html
<input type="hidden" name="form-name" value="Contact Form"/>
<input type="hidden" name="bot-field"/>
```

The above fields trigger the warnings...

```
   9:9  warning  A control must be associated with a text label  jsx-a11y/control-has-associated-label
  10:9  warning  A control must be associated with a text label  jsx-a11y/control-has-associated-label
```

**Comment by ljharb** [comment]
Indeed, there should be no warnings for any hidden input. Thanks for the report!

**Comment by jessebeach** [comment]
@hades200082 these examples are passing in the latest version 6.3.x.
